### DISCUSSION TITLE
Ultrawide monitor support

### DISCUSSION THREAD

**Discussion by VeraStray** [i3/i3#6140] [Q&A]
Hey, I might be looking for something that is outside of the core wm concept of always using maximum screen space. Because I have an ultra-wide (5120x1440) (49 inch), I never have a window maximized. The only app I ever run full screen is Davinci Resolve. Breaking my screen up into 3 columns make each column a nice width of 1700. So my ideal solution would be to have the first new window span in the center of the screen. A second new window would span to the right of the first window, and center the two. The third window would spawn  on the right again, pushing the other two over. Now there would be 3 columns across the screen. New windows would continue to spawn on the right, resizing windows to the left as usual. It's not clear to me if I could create this approach with i3.

**Comment by okraits** [comment]
You could spawn placeholder windows with `i3-msg open` and thus reduce the width a single window uses.

**Comment by orestisfl** [comment]
You can use randr virtual monitors for that, splitting your display in three monitors. Each monitor will have a different i3 workspace(s).

For reference, these commands split my 4K screen in two "columns":
```
xrandr --setmonitor 'DisplayPort-1~1' 1920/348x2160/392+0+0 DisplayPort-1
xrandr --setmonitor 'DisplayPort-1~2' 1920/349x2160/392+1920+0 none
```
**don't** copy and paste these, you need to update the monitor names and resolutions.

Some reading material:
- https://chipsenkbeil.com/notes/linux-virtual-monitors-with-xrandr/
- https://askubuntu.com/questions/150066/split-monitor-in-two/998435#998435

**Reply by orestisfl** [reply]
Note: you can use i3's `fullscreen global` command to make a window occupy the full physical monitor again.

**Reply by VeraStray** [reply]
That's a super interesting option I hadn't considered. Thank you!
